nodes/viz/viewer_draw_mk4.py

```diff
@@ -185,19 +185,20 @@ def view_3d_geom(context, args):
         bgl.glLineWidth(1)
 
     if config.draw_verts:
-        bgl.glPointSize(config.point_size)
-        if config.uniform_verts:
-            v_batch = batch_for_shader(config.v_shader, 'POINTS', {"pos": geom.v_vertices})
-            config.v_shader.bind()
-            config.v_shader.uniform_float("color", config.vector_color[0][0])
-        else:
-            v_batch = batch_for_shader(config.v_shader, 'POINTS', {"pos": geom.v_vertices, "color": geom.points_color})
-            config.v_shader.bind()
+        if len(geom.v_vertices)>0 and (len(geom.v_vertices[0])==3):
+            bgl.glPointSize(config.point_size)
```

Comment:
One more thing, if we are trying to bring all to pep8 =)

![image](https://github.com/nortikin/sverchok/assets/28003269/169cb6cd-b9e7-4ba2-8b3e-8c33d748e0e5)

Reply:
Sorry, I do not understand what you mean in this case? Should I remove something or check something else? )))

Reply:
```diff
- if len(geom.v_vertices)>0 and (len(geom.v_vertices[0])==3):
+ if geom.v_vertices and (len(geom.v_vertices[0])==3):
```

Reply:
NO! This is source of problems! 
geom.v_vertices is True when len of geom.v_vertices == 0 ! So blender will crash on
geom.v_vertices[0]
before calc len(geom.v_vertices[0]) !!!

Reply:
What I can see from the code is that `v_vertices` is a simple list. List, when it is empty, returns False. You can check with simple expression `bool([])`.

Reply:
I never heard about this trick! Thank you. Fixed.